Modelica/Media/Air/ReferenceMoistAir.mo

```diff
@@ -2443,7 +2443,7 @@ package ReferenceMoistAir
         end IceConstants;
 
         constant IceConstants Constants(
-          R_bar=8.314472,
+          R_bar=Modelica.Constants.R,
```

Comment:
This will change results slightly. 
If we still want to change it like this, we should not specify both R_s and MM, but instead calculate one of the two from
R_s=R/MM

Reply:
Reverted.